extensions/extensionservice/org.eclipse.smarthome.extensionservice.marketplace/src/main/java/org/eclipse/smarthome/extensionservice/marketplace/internal/model/Marketplace.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.extensionservice.marketplace.internal.model;
+
+/**
+ * This is the parent object that holds the category of the market.
+ *
+ * @author Kai Kreuzer - Initial contribution and API
+ */
+public class Marketplace {
+
+    /**
+     * The category of the marketplace
+     */
+    public Category[] categories;
+}
```

Comment:
We should add a new line at the end of the file.
Without the new line this will lead in noisy diffs in Git (and some quality tools also prefer new lines).
We should change the IDE to do it for us.